install/radomes.csv

```diff
@@ -1,25 +1,25 @@
 Make,Model,Serial,Mark,Start Date,End Date
 Leica Geosystems,LEIS,0220148020,MQZG,1999-09-27T00:00:00Z,2001-09-03T00:00:00Z
-Leica Geosystems,LEIS,11753,WGTN,1996-05-23T00:00:00Z,2005-03-21T01:59:59Z
-Leica Geosystems,LEIS,324,DUNT,1999-09-23T00:00:03Z,9999-01-01T00:00:00Z
+Leica Geosystems,LEIS,11753,WGTN,1999-11-11T00:00:00Z,2005-03-21T00:00:00Z
+Leica Geosystems,LEIS,324,DUNT,1999-09-23T00:00:00Z,9999-01-01T00:00:00Z
 Leica Geosystems,LEIS,332,WGTT,1999-12-14T00:00:00Z,2000-06-29T03:00:00Z
-Leica Geosystems,LEIS,332,TAKL,2001-07-11T00:00:00Z,2007-07-05T03:44:00Z
-Leica Geosystems,LEIS,333,WGTT,2000-06-29T03:00:01Z,9999-01-01T00:00:00Z
-Leica Geosystems,LEIS,405,LYTT,1999-11-15T00:00:03Z,9999-01-01T00:00:00Z
-Leica Geosystems,LEIS,CR13573,HOKI,2000-01-20T00:00:01Z,2003-09-05T22:56:00Z
-Thales,SCIS,0220063995,CNCL,2000-08-03T00:00:01Z,2016-02-29T20:26:00Z
-Thales,SCIS,0220066912,MTJO,2000-08-15T00:00:00Z,2011-02-07T22:33:00Z
+Leica Geosystems,LEIS,332,TAKL,2001-07-11T00:00:00Z,9999-01-01T00:00:00Z
+Leica Geosystems,LEIS,333,WGTT,2000-06-29T03:00:00Z,9999-01-01T00:00:00Z
+Leica Geosystems,LEIS,405,LYTT,1999-11-15T00:00:00Z,9999-01-01T00:00:00Z
+Leica Geosystems,LEIS,CR13573,HOKI,2000-01-20T00:00:00Z,2003-09-06T00:00:00Z
+Thales,SCIS,0220063995,CNCL,2000-08-03T00:00:00Z,9999-01-01T00:00:00Z
+Thales,SCIS,0220066912,MTJO,2000-08-15T00:00:00Z,9999-01-01T00:00:00Z
 Thales,SCIS,0220067545,WAKA,2000-11-02T00:00:00Z,2001-12-13T00:00:00Z
 Thales,SCIS,0220067548,CNCL,2000-02-05T00:00:00Z,2000-08-03T00:00:00Z
-Thales,SCIS,0220067548,WAKA,2002-05-31T02:44:30Z,2016-03-01T21:13:57Z
-Thales,SCIS,0220073544,NETT,2000-02-07T00:00:00Z,2008-03-18T04:00:00Z
+Thales,SCIS,0220067548,WAKA,2002-05-31T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
is there a reason why the undeploy time here is open (and not 2016-03-01 as it was?)